docs/source/get-started/package-managers.rst

```diff
@@ -126,3 +150,13 @@ For more complete examples, you can look at already existing recipes in the *Req
 
     spack dependents kokkos
 
+
+`Conda <https://https://anaconda.org/>`_
+----------------------------------------
+
+You may use the Conda package manager to install Kokkos:
+
+.. code-block::
+
+    conda install conda-forge::kokkos
```

Comment:
@carterbox as @dalg24 pointed out above: Could you spell the reason why you enable Serial and OpenMP?

Reply:
My intention during package bringup was to provide a fallback for users when they do not have NVIDIA hardware available. The channel culture (not official policy) of conda-forge is enable all the options and target broad user base. conda-forge has poor support for ROCm, so AMD GPUs are not an alternative option at this time. I don't think any of the maintainers of the kokkos feedstock have knowledge of accelerator frameworks for Intel.